src/ClassData.ts

```diff
@@ -52,7 +52,8 @@ var injectedFields: {[className: string]: {[fieldName: string]: [string, string]
     '$ws': ['Interfaces.IWebsock', 'null']
   },
   'Ljava/lang/Class;': {
-    '$cls': ['ClassData', 'null']
+    '$cls': ['ClassData', 'null'],
+    'signers': ['JVMTypes.java_lang_Object[]', 'null']
```

Comment:
Ah, good question! Our convention so far is to use `$` as a prefix for internal fields we add ourselves, but to omit the prefix when it is a HotSpot-dictated internal field (like `vmtarget`). So `signers` is fine.